create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+   
+    ```bash
+    terraform apply
+    ```
+
+    Sample output:
+    ```
+    tls_private_key.main: Creating...
+    kubernetes_namespace.flux_system: Creating...
+    github_repository.main: Creating...
+    tls_private_key.main: Creation complete after 2s [id=1d5ddec06b0f4daeea57d3a987029c1153ebcb21]
+    kubernetes_namespace.flux_system: Creation complete after 2s [id=flux-system]
+    kubectl_manifest.install["v1/serviceaccount/flux-system/source-controller"]: Creating...
+    kubectl_manifest.sync["kustomize.toolkit.fluxcd.io/v1beta1/kustomization/flux-system/flux-system"]: Creating...
+    kubectl_manifest.install["v1/serviceaccount/flux-system/helm-controller"]: Creating...
+    kubectl_manifest.install["networking.k8s.io/v1/networkpolicy/flux-system/allow-egress"]: Creating...
+    ...
+    ```
+
+# Next steps
+
+## Seeing the results
+If everything goes well the [DOKS cluster](https://cloud.digitalocean.com/kubernetes/clusters) should be up and running as well as [Flux CD](https://fluxcd.io):
+
+![DOKS state](content/img/doks_created.png)
+
+The terraform state file should be saved in your [DO Spaces](https://cloud.digitalocean.com/spaces) bucket so please go ahead and check it. It should look similar as seen in the picture down below:
+
+![DO Spaces Terraform state file](content/img/tf_state_s3.png)
+
+A new git repository should be created as well for you containing the Flux CD cluster resource files:
+
+![GIT repo state](content/img/flux_git_res.png)
+
+## Inspecting the cluster and applications state
+In order to inspect the Kubernetes cluster as well as the Flux CD state and getting information about various components we need to install a few tools like:
+
+1. `doctl` for DigitalOcean interaction (most of the tasks that can be done via the DO account web interface can be accomplished using the CLI version as well) 
+2. `kubectl` for Kubernetes interaction
+3. `flux` for Flux CD interaction
+   
+Having the above at hand will also help us create and manage the required Flux CD resources for example later on.
+
+### Doctl
+
+Can be installed for your specific OS and architecture from [here](https://github.com/digitalocean/doctl/releases). On MacOS we can use Homebrew to install it very easily like this:
+
+```bash
+brew info doctl
+brew install doctl
+```
+
+After installation we have to initialize it using the DigitalOcean personal token created at the beginning of this blueprint (when asked just paste it):
+
+```bash
+doctl auth init
+```
+
+Verify that the new auth context was added with:
+
+```bash
+doctl auth list
+```
+
+### Kubectl
+
+Please follow the specific steps for your OS and architecture from the official [site](https://kubernetes.io/docs/tasks/tools). On MacOS we can use Homebrew to install it very easily like this:
+
+```bash
+brew info kubectl
+brew install kubectl
+```
+
+Next we have to set kubectl context to point to our cluster created in this blueprint like this:
```

Comment:
```suggestion
Next we have to set `kubectl` context to point to our cluster created in this blueprint like this:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.